Agreements/Internal/MAW_MAE.txt

```diff
@@ -4,8 +4,8 @@ COPX:*:*:BARTN:EGCN:*:Scottish PC West:Scottish PC East:*:15000:BARTN5
 COPX:*:*:POL:EGNV:*:Scottish PC West:Scottish PC East:*:20000:POL
 COPX:*:*:POL:EGNT:*:Scottish PC West:Scottish PC East:*:24000:POL
 
-COPX:EGGP:*:NANTI:*:*:Scottish PC West:Scottish PC East:9000:*:NANTI
```

Comment:
Not on agreed levels - to be added i guess.

Reply:
Conflicting sources. I haven't ever worked this area and learnt what works... vMATS says climbing, but let me know what you'd prefer!